tests/unit/test_query_language.py

```diff
@@ -284,6 +284,11 @@ def test_dataset_setattr_rejects_invalid_variables(variable, error):
         Dataset().v = variable
 
 
+def test_dataset_setattr_gives_hint_for_accidental_tuple():
+    with pytest.raises(TypeError, match="trailing comma"):
+        Dataset().d = (patients.date_of_birth,)
```

Comment:
Should this be without the parentheses (and with a format skip comment) so it's more like an actually "accidental" tuple? It's all the same, really, so not too bothered either way

Reply:
Oh yes, thanks. I think I wrote it that way originally but left off the format skip and so Ruff reformatted it.